src/lambda_function.py

```diff
@@ -178,9 +179,9 @@ def lambda_handler(event, context):
         logger.info("parsing the following report: {}".format(key))
         csv_like_obj = s3client.get_object(Bucket=env_var['bucket'], Key=key)
         gen = CSVLineGenerator(csv_like_obj['Body'])
-        headers = csv.reader([gen.headers]).next()
+        headers = next(csv.reader([gen.headers]))
         for line in gen.stream_line():
             list_line = csv.reader([line])
-            shipper.add(_parse_file(headers, list_line.next(), event_time))
+            shipper.add(_parse_file(headers, next(list_line), event_time))
 
         shipper.flush()
```

Comment:
do we send the data to Logzio compressed in Gzip?

Reply:
No. We send lines from the csv file. (Bulks over HTTP).

Reply:
please add that it will send the data compressed by gzip.
Today customer pay to aws for each byte he send outside his cloud environment so it better to compress it to reduce the costs to the customer

Reply:
Done